compiler panic with aliases global

<!-- Please mention your platform and the programming language you are using Slint with -->

I am trying to compile the following Slint file to rust using `cargo run -p slint-compiler -- --format rust testcase.slint`:

```slint
global ResizeState {
    in property <length> handle-size: 10px;
}

component ResizeHandle {
    width <=> ResizeState.handle-size;
}

export component Resizer {
    ResizeHandle { 
        x: ResizeState.handle-size;
    }
}
```

The compiler panics without and error or warning with this input, unwrapping a `None` value.

Note that the C++ doesn't panic, but it also doesn't compile:

```
/tmp/.tmpjZsSGm.cpp:235:147: error: ‘const struct slint::cbindgen_private::Empty’ has no member named ‘width’
  235 |         case 2: return slint::private_api::convert_anonymous_rect(std::make_tuple(float(std::max<float>(0, 0)), float(self->resizehandle_3.root_4.width.get()), float(50), float(((self->root_window_1.height.get() - std::max<float>(0, 0)) /(float) 2))));
      |                                                                                                                                                   ^~~~~
```

--- a/internal/compiler/passes/binding_analysis.rs
+++ b/internal/compiler/passes/binding_analysis.rs
@@ -162,6 +162,12 @@ fn analyze_element(
     for nr in elem.borrow().accessibility_props.0.values() {
         process_property(&PropertyPath::from(nr.clone()), context, reverse_aliases, diag);
     }
+    if let Some(g) = elem.borrow().geometry_props.as_ref() {
+        process_property(&g.x.clone().into(), context, reverse_aliases, diag);
+        process_property(&g.y.clone().into(), context, reverse_aliases, diag);
+        process_property(&g.width.clone().into(), context, reverse_aliases, diag);
+        process_property(&g.height.clone().into(), context, reverse_aliases, diag);
+    }
 
     if let Some(component) = elem.borrow().enclosing_component.upgrade() {
         if Rc::ptr_eq(&component.root_element, elem) {

--- a/tests/cases/crashes/issue_4002_width_alias.slint
+++ b/tests/cases/crashes/issue_4002_width_alias.slint
@@ -1,0 +1,16 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+global ResizeState {
+    in property <length> handle-size: 10px;
+}
+
+component ResizeHandle {
+    width <=> ResizeState.handle-size;
+}
+
+export component Resizer {
+    ResizeHandle {
+        x: 50px;
+    }
+}

